src/lib/vasi-sync/src/sync.rs

```diff
@@ -182,33 +188,68 @@ impl<T: ?Sized> MutPtr<T> {
 
 unsafe impl<T: ?Sized> Send for MutPtr<T> where T: Send {}
 
+#[cfg(not(loom))]
+pub struct ConstPtr<T: ?Sized>(*const T);
+#[cfg(not(loom))]
+impl<T: ?Sized> ConstPtr<T> {
+    /// # Safety
+    ///
+    /// See `loom::cell::ConstPtr::deref`.
+    #[allow(clippy::mut_from_ref)]
```

Comment:
Why is this clippy lint needed? I don't see any `mut`?

Reply:
Oops, yeah copy-pasta